course_management/settings.py

```diff
@@ -109,12 +111,17 @@
 # Database
 # https://docs.djangoproject.com/en/4.2/ref/settings/#databases
 
-DATABASE_URL = os.getenv("DATABASE_URL", "sqlite:///db/db.sqlite3")
-db_config = dj_database_url.config(default=DATABASE_URL)
-
-DATABASES = {"default": db_config}
+# DATABASE_URL = os.getenv("DATABASE_URL", "sqlite:///db/db.sqlite3")
+# db_config = dj_database_url.config(default=DATABASE_URL)
 
+# DATABASES = {"default": db_config}
 
+DATABASES = {
```

Comment:
Can you please unroll this change? To run it locally with sqlite, use `export DATABASE_URL=sqlite:///db/db.sqlite3`

Reply:
Sorry, I will revert it.